### ISSUE TITLE
[1.3.x] classes not found trying to install

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#804]
```
WARN (4): E_WARNING: include_once(Base/bootstrap.php): failed to open stream: No such file or directory in bootstrap.php Zeile 21 
WARN (4): E_WARNING: include_once(): Failed opening 'Base/bootstrap.php' for inclusion (include_path='.:/usr/share/php:..:/domain/lib/vendor/:/domain/system/SecurityCenter/lib/vendor') in bootstrap.php Zeile 21 

Fatal error: Class 'AbstractMUVideo_Entity_Base_Collection' not found in /domain/modules/MUVideo/lib/MUVideo/Entity/Collection.php on line 33
```

**Comment by Portugao** [comment]
Seems not to be fixed.

in bootstrap line 21:
`` 
include_once 'Base/bootstrap.php';
```
the base classes for the entities are in a folder 'abstract' and named without abstract...

**Comment by Guite** [comment]
Thanks, I forgot that. Fixed in 665fe878b7f741db99774037312f36ec8bfb599b
